packages/frontend/src/features/metricsCatalog/components/MetricsCatalogPanel.tsx

```diff
@@ -316,6 +318,18 @@ export const MetricsCatalogPanel = () => {
                                     cursor: 'default',
                                     boxShadow:
                                         '0px -2px 0px 0px rgba(4, 4, 4, 0.04) inset',
+                                    '&:hover': {
+                                        cursor: 'pointer',
+                                    },
+                                }}
+                                onClick={() => {
+                                    // @ts-ignore
+                                    if (window.Pylon) {
+                                        // @ts-ignore
+                                        window.Pylon('show');
+                                    } else {
+                                        showIntercom();
```

Comment:
what if they don't have intercom? should we completely hide the button for them?

Reply:
This is the `Beta` badge, so I believe it should remain there but not do anything 🤔